tools/cbmc/proofs/HTTP/IotHttpsClient_AddHeader/IotHttpsClient_AddHeader_harness.c

```diff
@@ -45,15 +45,17 @@ void harness() {
   IotHttpsRequestHandle_t reqHandle = allocate_IotRequestHandle();
   if (reqHandle)
     __CPROVER_assume(is_valid_IotRequestHandle(reqHandle));
-  uint32_t nameLen;
-  uint32_t valueLen;
+
+  size_t nameLen;
+  size_t valueLen;
```

Comment:
You changed the type, can you confirm is this the type in the function prototype?  I see size_t used in the ReadHeader harness below.

Reply:
I fixed both this and ReadHeader harnesses. The function signature uses uint32_t, so I switched to that.